mail_browser_view/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Mail Browser View",
+    "summary": "Add 'View this email in browser' feature",
+    "version": "11.0.1.0.0",
+    "category": "Social Network",
+    "website": "https://github.com/OCA/social",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "mail",
+        "email_template_qweb",
```

Comment:
`email_template_qweb` I don't think this is a dependency, you don't care about how the body_html is rendered.
But you do care about styles and to make sure they are there I think we should depend on mail_inline_style.
If not, I think we should strongly encourage ppl to use it in the readme.